### ISSUE TITLE
Upload & Submitting Builds

### ISSUE THREAD

**Issue by ambikaroos** [department-of-veterans-affairs/va.gov-team#23049]
## Description
App builds need to be submitted for a full approval process

**Needed by**
- 2 weeks before launch

**Acceptance Criteria**
- [X] Build is uploaded to both stores
- [x] Build is approved

## Tasks & Status
**6/3**
- Submitted to both stores on 6/1
- Debugging

**5/24**
- We'll do this next week on 6/1

**5/14**
- Would be nice to have build a week ahead of time, but they are prioritizing our builds right now
- So we'd need a build by 6/1
- Could put a production build up and tell them we'd go live on 6/8, and once approved, its faster to put a second build up on 6/7 which won't trigger as much review

**older**
- Once a build is approved, is there then a toggle before it goes live?
